src/megaapi_impl.cpp

```diff
@@ -23073,6 +23081,45 @@ void MegaApiImpl::sendPendingRequests()
                                                            request->getNodeHandle()));
             break;
         }
+        case MegaRequest::TYPE_FETCH_GOOGLE_ADS:
+        {
+                MegaStringList* stringList = request->getMegaStringList();
+                std::vector<std::string> vectorString;
+                for (int i = 0; i < stringList->size(); i++)
+                {
+                    vectorString.push_back(stringList->get(i));
+                }
+
```

Comment:
Once you add an enumeration with the possible `adFlags`, we may want to add sanity checks here.

Reply:
Fixed